.prettierignore

```diff
@@ -2,4 +2,5 @@
 *.md
 docs
 *.css
-*.yml
\ No newline at end of file
+*.yml
+dist
```

Comment:
Added `dist` to ignore list since we don't want prettier churning our built code.